presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoDriver.java

```diff
@@ -406,19 +436,22 @@ public Connection connect(String url, Properties connectionProperties)
             return null;
         }
 
+        PrestoDriverUri uri = new PrestoDriverUri(url, driverProperties);
```

Comment:
`PrestoDriverUri` was fine before. But now it think it should be called somehow else.